integration/spark-common-cluster-test/src/test/scala/org/apache/spark/sql/common/util/QueryTest.scala

```diff
@@ -220,4 +218,74 @@ object QueryTest {
 
     return None
   }
+
+  /**
+   * execute the query by establishing the jdbc connection
+   *
+   * @param query
+   * @return
+   */
+  def executeQuery(dbName: String, query: String): List[Map[String, Any]] = {
+    Try {
+      val conn: Connection = if (System.getProperty("presto.jdbc.url") != null) {
+        createJdbcConnection(dbName, System.getProperty("presto.jdbc.url"))
+      } else {
+        createJdbcConnection(dbName, "localhost:8086")
+      }
+      val statement = conn.createStatement()
+      val result: ResultSet = statement.executeQuery(query)
+      convertResultSetToList(result)
+    } match {
+      case Success(result) => result
+      case Failure(jdbcException) =>
+        throw jdbcException
+    }
+  }
+
+  /**
+   * Creates a JDBC Client to connect CarbonData to Presto
+   *
+   * @return
+   */
+  private def createJdbcConnection(dbName: String, url: String) = {
+    val JDBC_DRIVER = "com.facebook.presto.jdbc.PrestoDriver"
+    var DB_URL : String = null
+    if (dbName == null) {
```

Comment:
better to add  dbName.equals("") or use StringUtils.isEmpty

Reply:
ok.done